frontend/projects/ui/src/app/modals/generic-input/generic-input.component.ts

```diff
@@ -1,20 +1,26 @@
 import { Component, Input, ViewChild } from '@angular/core'
 import { ModalController, IonicSafeString, IonInput } from '@ionic/angular'
 import { getErrorMessage } from '@start9labs/shared'
+import { MaskPipe } from 'src/app/pipes/mask/mask.pipe'
 
 @Component({
   selector: 'generic-input',
   templateUrl: './generic-input.component.html',
   styleUrls: ['./generic-input.component.scss'],
+  providers: [MaskPipe],
```

Comment:
You shouldn't need this

Reply:
Why?

Reply:
MaskPipe was needed here in order to use it in the component rather than the template